src/components/combobox/combobox.e2e.ts

```diff
@@ -481,21 +471,188 @@ describe("calcite-combobox", () => {
+    let page: E2EPage;
+    let label;
+    let popper;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(
+        html`
+          <calcite-label id="label">
+            <span class="label-text">Yeah<span>
+            <calcite-combobox id="parentOne">
+              <calcite-combobox-item id="one" icon="banana" value="one" text-label="One"></calcite-combobox-item>
+              <calcite-combobox-item id="two" icon="beaker" value="two" text-label="Two"></calcite-combobox-item>
+              <calcite-combobox-item id="three" value="three" text-label="Three"></calcite-combobox-item>
+            </calcite-combobox>
+          </calcite-label>
+        `
+      );
+      const inputEl = await page.find(`#parentOne >>> input`);
+      await inputEl.focus();
+      await page.waitForChanges();
+      label = await page.find("#label");
+      popper = await page.find(`#parentOne >>> .popper-container`);
+    });
+
+    it("label click focuses on the combobox, does not open", async () => {
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+      const visible = await popper.isVisible();
+      expect(visible).toBe(false);
+
+      await label.click();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+      await popper.isVisible();
+      expect(visible).toBe(false);
+    });
+
+    it("label click closes the listbox if it's open", async () => {
+      expect(await page.evaluate(() => document.activeElement.id)).toBe("parentOne");
+      expect(await popper.isVisible()).toBe(false);
+
+      await page.keyboard.press("ArrowDown");
+      await page.waitForChanges();
+      expect(await popper.isVisible()).toBe(true);
+
+      await label.click();
+      await page.waitForChanges();
+      expect(await popper.isVisible()).toBe(false);
+    });
+  });
+
   describe("keyboard navigation", () => {
-    it("should cycle through items on up/down arrows", async () => {
-      const page = await newE2EPage();
+    let page: E2EPage;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
       await page.setContent(
         html`
-          <calcite-combobox>
-            <calcite-combobox-item id="one" value="one" text-label="one"></calcite-combobox-item>
-            <calcite-combobox-item id="two" value="two" text-label="two"></calcite-combobox-item>
+          <calcite-combobox id="parentOne">
+            <calcite-combobox-item id="one" value="one" label="one"></calcite-combobox-item>
+            <calcite-combobox-item id="two" value="two" label="two"></calcite-combobox-item>
+            <calcite-combobox-item-group label="Last Item">
+              <calcite-combobox-item id="three" value="three" label="three"></calcite-combobox-item>
+            </calcite-combobox-item-group>
+          </calcite-combobox>
+          <calcite-combobox id="parentTwo">
```

Comment:
Can the tests use a single combobox or why is a sibling combobox needed?

Reply:
To test tabbing behavior and the second combobox items are selected to make it easier to test chip interaction.

Reply:
Tabbing does not depend on another combobox being present though. The chip interaction is a different test and it would be better for maintainability to separate it.